### DISCUSSION TITLE
Permitted process open only in backgound

### DISCUSSION THREAD

**Discussion by guysiTomax** [SafeExamBrowser/seb-win-refactoring#409] [Q&A]
Hi,

we are using 3.3.1 and we have permitted process of calculator which is shown on the taskbar,
however when clicked it always opened in background  and can only be accessed once the user quit SEB
this is the configuration :
![image](https://user-images.githubusercontent.com/94598094/168473988-cf0fc6c1-a9c8-4e29-bb58-d0d6b3b5542f.png)

thanks much,
Guy

**Comment by dbuechel** [comment]
SEB is not compatible with UWP apps (and the Windows calculator is currently a UWP app). Thus you'd either need to use another calculator application or e.g. use an online calculator.

**Reply by guysiTomax** [reply]
Thanks, actually we managed to use win 7 calculator by changing the exe filename
